3dhubs_amsterdam_python_flask.html

```diff
@@ -0,0 +1,64 @@
+---
+title:  Python Developer
+company: 3D Hubs
+url: https://www.3dhubs.com/
+location: Amsterdam, the Netherlands
+contract: temporary but can be converted to permanent 
+contact:
+    name: Mrs A. Cojanu
```

Comment:
I think this would be better if you wrote <firstname> <lastname>, e.g. Alina Cojanu... unless of course the company really does prefer to use formal names.